nilearn/plotting/tests/test_displays.py

```diff
@@ -15,7 +15,7 @@
 from nilearn._utils.extmath import fast_abs_percentile
 
 
-###############################################################################
+#
```

Comment:
Interesting "fix", is that autopep8 doing it? Arguably a better one would be to use `# ######` i.e. a space after the first hash.

Full disclosure, that may be just me but I am not a massive fan of these separating lines.